### PR TITLE
Create calculate and operate modules

### PR DESCRIPTION
**Pull request by bettercallazamat**
In this PR I implemented two JavaScript modules: calculate.js and operate.js which will be responsible for all calculations.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        ? numTwo * -1
+        : numOne * -1;
+      break;
+
+    case 'AC':
+      result = 0;
+      break;
+
+    case '.':
+      result = 0.0;
```
Comment (bellom) [inline src/logic/operate.js]: ### Heads-Up:

If the `buttonName`  = `.` we should return `0.`

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import operate from './operate';
+
+const calculate = (buttonName, data) => {
+  const { next, operation, total } = data;
+  const result = operate(
```
Comment (bellom) [inline src/logic/calculate.js:5]: I believe you want to add the conditions in the next milestone.

Heads-Up:
Make sure to return an error message when a number is divided by 0. `(e.g. 4 / 0)`

**Review by bellom — APPROVED** [review]
## Hi @bettercallazamat  👏  

#### Awesome job so far!! :+1: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

File: src/logic/operate.js (hunk 1 above)
Comment (bettercallazamat) [reply src/logic/operate.js]: Solved a different way.

**Pull request merged**